### DISCUSSION TITLE
upgraded to Prisma 4.11.0

### DISCUSSION THREAD

**Discussion by RTestM** [prisma/prisma#18565] [Q&A]
I have upgraded to Prisma 4.11.0, and my MySQL version is 8.0.31. After upgrading, we are getting some npm errors. Like `npm ERR! command sh -c -- prisma migrate deploy`

**Comment by nurul3101** [comment]
Hello @RTestM 👋 

Can you please share the entire error stack trace?
Does the error go away if you downgrade to Prisma `4.10.0`?
Can you please include DEBUG environment variable and check if you get some additional information? ([Reference](https://www.prisma.io/docs/guides/database/troubleshooting-orm/creating-bug-reports#include-logging-and-debugging-output))

**Reply by nurul3101** [reply]
Are you still facing the issue> It would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏
